crates/optimism/rpc/src/lib.rs

```diff
@@ -12,6 +12,7 @@ pub mod engine;
 pub mod error;
 pub mod eth;
 pub mod historical;
+pub mod historical_proofs;
```

Comment:
can we pls move this into the eth module as submodule and just call it proofs? the name historical proofs makes me think of bedrock proofs the more I hear it...

Reply:
good idea! updated